db/migrate/20210618104853_add_code_index_to_experiments.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class AddCodeIndexToExperiments < ActiveRecord::Migration[6.1]
+  def up
+    ActiveRecord::Base.connection.execute(
+      "CREATE INDEX index_experiments_on_experiment_code ON experiments using gin (('EX'::text || id) gin_trgm_ops);"
```

Comment:
Can you use 'add_gin_index_without_tags' here?

Reply:
It would try and interpolate `('EX'::text || id) ` into the index name, so I guess not, unless I modify the helper. It would also run the trim_html_tags on it, which is unnecessary.